omniduct/filesystems/s3.py

```diff
@@ -189,11 +189,27 @@ def _dir(self, path):
     #         if pattern is None or pattern.match(k.key[len(path_prefix):]):
     #             yield k.key
 
-    def _mkdir(self, path, recursive):
-        raise NotImplementedError("Support for S3 write operations has yet to be implemented.")
+    def _mkdir(self, path, recursive, exist_ok):
+        if not path.endswith('/'):
+            path += '/'
+        if not self._exists(path):
+            self._client.put_object(Bucket=self.bucket, Key=self._path(path))
```

Comment:
You don't need to do `self._path` here, since `path` has already been sanitised in `FileSystemClient.mkdir`.

Reply:
didn't get you. Can you give me an example how to pass this param?

Reply:
ok... you mean to say that we can directly pass the path variable?

Reply:
fixed it.